lnbits/wallets/fake.py

```diff
@@ -31,8 +31,8 @@
 
 class FakeWallet(Wallet):
     queue: asyncio.Queue = asyncio.Queue(0)
-    payment_secrets: Dict[str, str] = dict()
-    paid_invoices: Set[str] = set()
+    payment_secrets: ClassVar[Dict[str, str]] = {}
+    paid_invoices: ClassVar[Set[str]] = set()
```

Comment:
how was decided that only these two vars are `ClassVar` while the other not?

Reply:
ClassVar marks which are mutable object
https://docs.astral.sh/ruff/rules/mutable-class-default/